symfony.lock

```diff
@@ -1,4 +1,96 @@
 {
```

Comment:
Could you pls ansert me, why are you pushing `symfony.lock` file :thinking: ? Is there some reason for this?